pol-core/bscript/compiler/ast/ClassDeclaration.h

```diff
@@ -30,8 +31,15 @@ class ClassDeclaration : public Node
   // Owned by top_level_statements
   Node* class_body;
 
-  // Will be null if class has no constructor declared.
-  const std::shared_ptr<FunctionLink> constructor;
+  // Will have no function linked if class has no constructor defined.
+  const std::shared_ptr<FunctionLink> constructor_link;
+
+  // Filled by SemanticAnalyzer with functions owned in
+  // CompilerWorkspace::user_functions.
+  std::vector<std::shared_ptr<ClassLink>> base_class_links;
```

Comment:
comment is wrong, done by generated function builder (was done in sem analyzer prior to refactor commit)